### PR TITLE
additional tests for arguments parser

### PR DESCRIPTION
**Pull request by logogin**

### CODE REVIEW

File: test/test_arguments.py [hunk 1]
```diff
@@ -48,7 +49,93 @@
...
+    _test_args(test_params, expected_params, arguments.add_inference_args)
+
+
+def _test_args(test_params, expected_params, args_func):
     test_parser = argparse.ArgumentParser()
-    arguments.add_io_args(test_parser)
+    args_func(test_parser)
     parsed_params = test_parser.parse_args(test_params.split())
-    assert dict(vars(parsed_params)) == expected_params
+    assert dict(vars(parsed_params)) == expected_params
```
Comment (fhieber) [inline test/test_arguments.py]: File is missing a newline at the end

File: test/test_attention.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def test_get_context_and_attention_probs():
 
     # data
     source_nd = mx.nd.random_normal(shape=(batch_size, seq_len, num_hidden))
-    source_length_np = np.random.random_integers(1, seq_len, (batch_size,))
```
Comment (fhieber) [inline test/test_attention.py]: why that change?

File: test/test_arguments.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 
 import pytest
 
+import sockeye.constants as const
```
Comment (fhieber) [inline test/test_arguments.py]: we have a convention to import constants module as C (which is not ideal for code checking, I know, but we could keep it consistent here)

**Comment by fhieber** [comment]
@logogin ping

File: test/test_attention.py (hunk 2 above)
Comment (logogin) [inline test/test_attention.py]: [numpy.random.random_integers](https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.random_integers.html) is deprecated

**Comment by logogin** [comment]
rebased

File: test/test_attention.py (hunk 2 above)
Comment (fhieber) [inline test/test_attention.py]: I see, makes sense!

**Pull request merged**